index.html

```diff
@@ -255,14 +288,20 @@ <h5>Skills</h5>
         </a>
       </li>
     </ul>
-  </ul>
+  </div>
+  </div>
    </section>
-<section id="contact">
+
+<!-- Contact -->
+<section class="contact">
   <div class="contact_form">
     <h2>Contact Me</h2>
     <p class="contact_desc">
-    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it!
-  </p>
+    If you have an application you are interested in developing, a
+    feature that you need built or a project that needs coding. I'd love
+    to help with it!</p>
+  </div>
+  <div class="form">
   <form action="https://formspree.io/f/mlekylyd" method="post" class="form">
     <ul class="form_input">
       <li>
```

Comment:
- Good job done so far 👏, but there are some issues to be fixed in the contact form section of your project, I have marked these parts in the image below.

Your project:
- The text in this section should be centered in the middle and not spanning across the whole width of the section, please fix this 🙏 
- Please give the input forms some more width just as it is in the design template 😄 
- The contact form button should be aligned in the middle, just as it is in the design template, please fix this 👍 👍
![eyeyey](https://github.com/katarighe/my-portfolio/assets/73607512/54628130-cd96-451c-9dfd-1c99193b0551)

Design template:
![rgawrgaw](https://github.com/katarighe/my-portfolio/assets/73607512/31aec5b3-8573-4942-ba27-6ac8c08034e4)